jenkins-integration/jenkins-jobs/scenarios/oss-puppetserver-5.0.x/Jenkinsfile

```diff
@@ -0,0 +1,26 @@
+node {
+    checkout scm
+    pipeline = load 'jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy'
+}
+
+pipeline.single_pipeline([
+        job_name: 'oss-5.0.x',
```

Comment:
Yeah, that's a good point.  I suppose we'll probably only have one "5.x" vs. one "master" branch active at a time so maybe just making this one a "5.x" instead of a "5.0.x" would be better.  I'll work up that change.